__tests__/components/AppLayout/SidebarItems/SettingsNavigationItem.comprehensive.test.tsx

```
@@ -88,141 +116,155 @@ const mockUseSettingsNormal = () => ({
-  it("should show updating state during update", async () => {
-    const { useSettings } = require("@/hooks/settings");
-    const { updateSettings } = require("@/lib/services/settings.service");
-    
+  it('should show updating state during update', async () => {
+    const { useSettings } = require('@/hooks/settings');
+    const { updateSettings } = require('@/lib/services/settings.service');
+
     useSettings.mockImplementation(mockUseSettingsNormal);
-    
+
     let resolveUpdate: (value: any) => void;
```

ESLint

[error] Unexpected any. Specify a different type.

(@typescript-eslint/no-explicit-any)